### PR TITLE
Cut file

### PR DESCRIPTION
**Pull request by lisabbasil**

### CODE REVIEW

File: TTHAnalysis/python/plotter/susy-1lep/1l_basicCutsFullCutFlow.txt [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
 Selected: Selected == 1
 
-Trigger: !isData || (isData && (HLT_EleHT350 || HLT_EleHT400 || HLT_Ele105 || HLT_MuHT400 || HLT_MuHT350))
+# Fire at least one trigger
+Trigger: !isData || (HLT_EleHT350 || HLT_EleHT400 || HLT_Ele105 || HLT_MuHT400 || HLT_MuHT350)
 
-# use the following line to pick leptons only from the corresponding PD
-XOR: !isData || (isData && ((PD_SingleEle && nEl==1) || (PD_SingleMu && nMu==1)))
+# Prevent double counting from different datasets
+Trigger XOR: !isData || ((PD_SingleEle && (HLT_EleHT350 || HLT_EleHT400 || HLT_Ele105)) || (PD_SingleMu && (HLT_MuHT400 || HLT_MuHT350) && !((HLT_EleHT350 || HLT_EleHT400 || HLT_Ele105))))
```
Comment (kirschen) [inline TTHAnalysis/python/plotter/susy-1lep/1l_basicCutsFullCutFlow.txt:14]: Logic looks good to me. 
However, there are triggers missing (Ele105 became prescaled; Ele115 need to be ORed as well; want to also add the Ele50_Jet165-trigger etc.) + we will have to update this again with the MET dataset.

Comment (lisabbasil) [reply TTHAnalysis/python/plotter/susy-1lep/1l_basicCutsFullCutFlow.txt:14]: I simply took the triggers that were already present in the cutfile and changed the logic, such that it easily can be extended to more triggers and more datasets. I trust on you to present the final list of triggers that will be used.

Comment (clseitz) [reply TTHAnalysis/python/plotter/susy-1lep/1l_basicCutsFullCutFlow.txt:14]: Once we have all the trigger settled, maybe we could just add 3 variables to the FT that represent the ele, mu, met triggers?

Comment (kirschen) [reply TTHAnalysis/python/plotter/susy-1lep/1l_basicCutsFullCutFlow.txt:14]: Agreed. I was wondering as well, how much of a difference it makes for the performance of plotting if we have a more gigantic string than necessary.

I'll merge this PR and then do the rest on top of it.

**Pull request merged**